formfactory/templatetags/formfactory_tags.py

```diff
@@ -8,5 +9,5 @@
     "formfactory/inclusion_tags/form_detail.html", takes_context=True
 )
 def render_form(context, obj):
-    context["object"] = obj
+    context["object"] = Form.objects.get_object_or_404(slug=obj)
```

Comment:
You'll need to detect weather the obj passed is a slug or object and than apply the appropriate logic. Docs and tests missing.